tests/tezos-nix-binaries.nix

```diff
@@ -19,28 +19,28 @@ in import "${nixpkgs}/nixos/tests/make-test-python.nix" ({ ... }:
   };
 
   testScript = ''
-    tezos_accuser = "${tezos-accuser-013-PtJakart}/bin/tezos-accuser-013-PtJakart"
+    octez_accuser = "${octez-accuser-PtKathma}/bin/octez-accuser-PtKathma"
     tezos_admin_client = "${tezos-admin-client}/bin/tezos-admin-client"
-    tezos_baker = "${tezos-baker-013-PtJakart}/bin/tezos-baker-013-PtJakart"
-    tezos_client = (
-        "${tezos-client}/bin/tezos-client"
+    octez_baker = "${octez-baker-PtKathma}/bin/octez-baker-PtKathma"
+    octez_client = (
+        "${octez-client}/bin/octez-client"
     )
-    tezos_node = "${tezos-node}/bin/tezos-node"
-    tezos_signer = (
-        "${tezos-signer}/bin/tezos-signer"
+    octez_node = "${octez-node}/bin/octez-node"
+    octez_signer = (
+        "${octez-signer}/bin/octez-signer"
     )
-    tezos_codec = "${tezos-codec}/bin/tezos-codec"
+    octez_codec = "${octez-codec}/bin/octez-codec"
     openssl = "${pkgs.openssl.bin}/bin/openssl"
     host_key = "${./host.key}"
     host_cert = "${./host.cert}"
     binaries = [
-        tezos_accuser,
+        octez_accuser,
         tezos_admin_client,
-        tezos_baker,
-        tezos_client,
-        tezos_node,
-        tezos_signer,
-        tezos_codec,
+        octez_baker,
+        octez_client,
+        octez_node,
+        octez_signer,
+        octez_codec,
```

Comment:
Most changes in this file don't seem to belong to this commit (f4f281cf49b1fd5eef075b6dc4791f6a938c0f92), as they are not about jakarta's removal, but rather due to the renaming of the binaries.

Please try to move them to a more appropriate commit